src/components/Reports/LostTime/AddLostTime.jsx

```diff
@@ -285,15 +288,15 @@ const AddLostTime = props => {
   };
 
   return (
-    <Modal isOpen={props.isOpen} toggle={props.toggle}>
-      <ModalHeader toggle={props.toggle}>
+    <Modal isOpen={props.isOpen} toggle={props.toggle} className={darkMode ? 'text-light dark-mode' : ''}>
+      <ModalHeader toggle={props.toggle} className={darkMode ? 'bg-space-cadet text-light' : ''}>
         Add Lost Time
       </ModalHeader>
-      <ModalBody>
+      <ModalBody className={darkMode ? 'bg-yinmn-blue' : ''}>
         <Form>
           <FormGroup>
-            <Label>Type</Label><br/>
-            <div className='type-container'>
+            <Label for="entryType" className={darkMode ? 'text-light' : ''} >Type</Label><br/>
```

Comment:
Since we are going to be repeating this line of code throughout this file, it is better to declare a variable like `const fontColor = darkMode ? 'text-light' : ' ';` and replace it within our classnames like so `className={fontColor}`